queries.sql

```diff
@@ -100,3 +100,53 @@ ON animals.owner_id = owners.id
 GROUP BY owners.full_name
 ORDER BY COUNT(name) DESC
 LIMIT 1
+
+ALTER TABLE visits
+ADD COLUMN date_of_visit date
+
+SELECT name,animal_id,vet_id,date_of_visit FROM visits JOIN animals
+ON animal_id = id
+WHERE vet_id = 1 AND date_of_visit = (SELECT MIN(date_of_visit) FROM visits WHERE vet_id = 1);
+
+SELECT name,animal_id,vet_id FROM visits JOIN animals
+ON animal_id = id
+WHERE vet_id = 3 
```

Comment:
- [ ] Lastly, you are expected to provide only screenshots for the under-listed queries and NOT screenshots of the **table creation**, and **data insertions**. Kindly update your pull request screenshots with ONLY that of the below queries.  As I mentioned previously, please use each question below as a label for your screenshots and query code. This way, it will make it easy for any to know what your code does. 

### Write queries to answer the following:

- Who was the last animal seen by William Tatcher?
- How many different animals did Stephanie Mendez see?
- List all vets and their specialties, including vets with no specialties.
- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
- What animal has the most visits to vets?
- Who was Maisy Smith's first visit?
- Details for most recent visit: animal information, vet information, and date of visit.
- How many visits were with a vet that did not specialize in that animal's species?
- What specialty should Maisy Smith consider getting? Look for the species she gets the most.

**_Take a screenshot of the results of your queries._**